ginga/main.py

```diff
@@ -292,10 +297,10 @@ def main(self, options, args):
 
         if toolkit == 'choose':
             try:
-                from ginga.qtw import QtHelp
+                from ginga.qtw import QtHelp  # noqa
```

Comment:
Technically it's really flake8 that's complaining. In this case, about unused import.